_data/backup.yml

```diff
@@ -4,10 +4,9 @@ websites:
     img: icloud.png
     tfa: Yes
     sms: Yes
+    phone: Yes
     software: Yes
-    exceptions:
-        text: "See https://support.apple.com/HT202656 for a list of supported SMS carriers."
```

Comment:
The exception still applies to SMS based 2FA.

Reply:
I just spent 20min on the phone with Apple support to clarify this and he did not see any explicit mention 2FA is restricted to some carriers the way 2SV is.